### PR TITLE
Feature/add partner path

### PR DESCRIPTION
**Pull request by jdmedlock**
Add a description of our partnerships and the general training services they provide.

### CODE REVIEW

File: docs/gettingstarted/gettinghelp.md [hunk 1]
```diff
@@ -15,43 +15,42 @@ There's no shame in asking for help and at Chingu we believe:
...
-    
+
 ### Chingu Questions & Issues
-    
+  
 If you need assistance from our Admin Team you can open up a support ticket
 by clicking on the [#open-support-ticket](https://discord.com/channels/330284646283608064/1105911757177888908) channel. Make sure that you follow these [guidelines](#how-to-ask-a-good-question) to help us understand and quickly resolve your question.
 
-Another good resource it the [#community-chat](https://discord.com/channels/330284646283608064/578606247448936448) is the best place to ask questions 
+Another good resource it the [#community-chat](https://discord.com/channels/330284646283608064/578606247448936448) is the best place to ask questions
```
Comment (cherylli) [inline docs/gettingstarted/gettinghelp.md:42]:
```suggestion
Another good resource is the [#community-chat](https://discord.com/channels/330284646283608064/578606247448936448) is the best place to ask questions
```

should be is?

**Review by cherylli — APPROVED** [review]

File: docs/gettingstarted/communitystds.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # Community Standards
 
-We view this as an obligation to all of our Members. Any Members violating 
-the Community Standards will be removed from the community. Violations should 
-be reported to [support@chingu.io](mailto:support@chingu.io) and will be kept 
+We view this as an obligation to all of our Members. Any Members violating
+the Community Standards will be removed from the community. Violations should
```
Comment (ellrub) [inline docs/gettingstarted/communitystds.md:4]: not the ticket system?

File: docs/gettingstarted/partners.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+![Chingu and its Partners](./assets/Chingu_partners.jpg)
+
+Chingu's mission is to help our members transform what they've learned into the experience needed to get jobs. As such, we don't provide extensive training like you would find in school, bootcamps, online training courses, and self-study.
+
+But, sometimes our members need additional training while in Chingu. To help we've partnered with the following organizations to help when you need training:
+
+* [The Job Hackers AgileMBA](https://www.thejobhackers.org/participant/) is a program to help those in Agile roles, Product Owners and Scrum Master, get the training needed to be effective in them.
+
+* [Scrimba Pro Plan](https://v2.scrimba.com/?via=chingu) provides high quality technical training at a 20% discount to Chingus.
```
Comment (ellrub) [inline docs/gettingstarted/partners.md:11]: which will guarantee atleast 20% off

File: docs/gettingstarted/whattodofirst.md [hunk 4]
```diff
@@ -71,43 +71,41 @@ graph LR
...
 
-When you are done, submit a [Solo Project Completion form](https://docs.google.com/forms/d/e/1FAIpQLSdN3dN7OjzppDCBNIsIf-bOJC_XAlQ9eiPcYQx_m6DfhaiRmA/viewform) so we can evaluate your project. 
+When you are done, submit a [Solo Project Completion form](https://docs.google.com/forms/d/e/1FAIpQLSdN3dN7OjzppDCBNIsIf-bOJC_XAlQ9eiPcYQx_m6DfhaiRmA/viewform) so we can evaluate your project.
 (Note: please use the same email as your application form)
-    
-One of our Facilitators will evaluate your Solo Project and will DM you in 
-Discord with feedback you can use! They will look at both your code as well 
-as the running app so you'll get 360-degree feedback. 
+
+One of our Facilitators will evaluate your Solo Project and will DM you in
```
Comment (ellrub) [inline docs/gettingstarted/whattodofirst.md:91]: One of our evaluators

File: docs/gettingstarted/communitystds.md (hunk 2 above)
Comment (jdmedlock) [reply docs/gettingstarted/communitystds.md:4]: Agreed. Changed to direct them to open a support ticket.

File: docs/gettingstarted/whattodofirst.md (hunk 4 above)
Comment (jdmedlock) [reply docs/gettingstarted/whattodofirst.md:91]: Agreed. Changed to Evaluators

File: docs/gettingstarted/partners.md (hunk 3 above)
Comment (jdmedlock) [reply docs/gettingstarted/partners.md:11]: Agreed. updated

**Pull request merged**